src/views/CurrencyExchange.vue

```diff
@@ -46,41 +46,49 @@
-        startingCurrency: '',
-        endingCurrency: '',
-        amount: 0,
-        conversionRate: null,
-        currencies : [ "AUD","BGN","BRL","CAD","CHF","CNY","CZK","DKK","EUR","GBP","HKD","HRK","HUF","IDR","ILS","INR","ISK","JPY","KRW","MXN","MYR","NOK","NZD","PHP","PLN","RON","RUB","SEK","SGD","THB","TRY","USD","ZAR"]
-      };
-    },
-    computed: {
-      convertedAmount() {
-        if (this.conversionRate && this.amount) {
-          return (this.amount * this.conversionRate).toFixed(2);
-        } else {
-          return '0.00';
-        }
+  data() {
+    return {
+      startingCurrency: '', // Modify this to be a string variable
+      endingCurrency: '',
+      amount: 0,
+      conversionRate: null,
+      currencies : [ "AUD","BGN","BRL","CAD","CHF","CNY","CZK","DKK","EUR","GBP","HKD","HRK","HUF","IDR","ILS","INR","ISK","JPY","KRW","MXN","MYR","NOK","NZD","PHP","PLN","RON","RUB","SEK","SGD","THB","TRY","USD","ZAR"]
+    };
+  },
+  computed: {
+    convertedAmount() {
+      if (this.conversionRate && this.amount) {
+        return (this.amount * this.conversionRate).toFixed(2);
+      } else {
+        return '0.00';
       }
-    },
-    methods: {
-        async convertCurrency() {
-        console.log('Convert currency button clicked');
-        console.log('Starting currency:', this.startingCurrency);
-        console.log('Ending currency:', this.endingCurrency);
-        console.log('Amount:', this.amount);
+    }
+  },
+  methods: {
+    async convertCurrency() {
+  console.log('Convert currency button clicked');
+  console.log('Starting currency:', this.startingCurrency);
+  console.log('Ending currency:', this.endingCurrency);
+  console.log('Amount:', this.amount);
 
   if (this.startingCurrency && this.endingCurrency && this.amount) {
     try {
-      const response = await axios.get(`https://api.freecurrencyapi.com/v1/latest?apikey=${this.$store.getters.getCurrencyApiKey}&currencies=${this.endingCurrency}`);
+      const response = await axios.get(`https://open.er-api.com/v6/latest/${this.startingCurrency}`);
       console.log('Response:', response);
 
-      if (response.data && response.data.data) {
-        const rates = response.data.data;
+      if (response.data && response.data.rates) {
+        const rates = response.data.rates;
+
         if (this.endingCurrency in rates) {
-          this.conversionRate = rates[this.endingCurrency];
-          console.log('Conversion rate:', this.conversionRate);
+          const conversionRate = rates[this.endingCurrency];
+          console.log('Conversion rate:', conversionRate);
+
+          // Calculate converted amount
+          const convertedAmount = this.amount * conversionRate;
+
+          this.conversionRate = conversionRate;
+          this.$store.commit('setAlertStatus', 'alert-success');
+          this.$store.commit('setAlertMessage', `Conversion successful! Converted amount: ${convertedAmount.toFixed(2)} ${this.endingCurrency}`);
```

Comment:
Honsetly I'm not a huge fan of using the alert UI this way, especially when the result is shown another place on the page. The alert is more to show the user warnings or errors like you do [here](https://github.com/WSU-4110/Travel-Companion/blob/a3ec1c849ddb0ea534c13ff5185889af92c96b1f/src/views/CurrencyExchange.vue#L104-L105). I'd also maybe recommend at some point adding a spinner or something when waiting for the response kinda like [this](https://github.com/WSU-4110/Travel-Companion/blob/development/src/views/SignInSignUpView.vue#L40-L46), although it's not entierly necessary since the call is pretty fast.

Reply:
More than willing to take out the success alerts, I put them in while having issues populating the output text field but i got that all fixed so i can remove these